packages/commons/src/config/EnvironmentVariablesService.ts

```diff
@@ -72,6 +84,28 @@ class EnvironmentVariablesService extends ConfigService {
     return truthyValues.includes(value.toLowerCase());
   }
 
+  /**
+   * It parses the key/value data present in the _X_AMZN_TRACE_ID environment variable
+   * and returns it as an object when available.
+   */
+  private getXrayTraceData(): Record<string, string> | undefined {
+    const xRayTraceEnv = this.get(this.xRayTraceIdVariable);
+
+    if (xRayTraceEnv === '') return undefined;
+
+    if (!xRayTraceEnv.includes('=')) return { Root: xRayTraceEnv };
```

Comment:
Note: I'm checking for `=` instead of `;` so that an expression like `Root=123` still parses correctly.